src/app/rundown-view/components/mini-shelf/mini-shelf.component.scss

```diff
@@ -31,37 +31,37 @@
   .media-unavailable {
     background: var(--transparent-and-red-stripes);
   }
+}
 
-  &__thumbnail {
-    display: flex;
-    padding: 0;
-    margin: 0;
-    width: 100%;
-    height: 60px;
-    object-fit: cover;
-    object-position: 50% 60%;
-  }
-
-  &__note {
-    display: flex;
-    padding: 2px;
-    height: 18px;
-    align-items: center;
-    justify-content: space-between;
-    text-shadow: var(--black-color) 0 0 2px;
-    font-weight: 450;
-    font-size: 12px;
-  }
+.note {
+  display: flex;
+  padding: 2px;
+  height: 18px;
+  align-items: center;
+  justify-content: space-between;
+  text-shadow: var(--black-color) 0 0 2px;
+  font-weight: 450;
+  font-size: 12px;
```

Comment:
font size should be in rem. The equivalent value is `0.75rem`.

Reply:
Updated